### ISSUE TITLE
[accessibility] Documentation search field doesn't have a label

### ISSUE THREAD

**Issue by Lausselloic** [twbs/bootstrap#21842]
Hello,

Documentation search field is missing label. Maybe just put a title attribute "Search in the documentation" will help 

http://v4-alpha.getbootstrap.com/getting-started/introduction/

Let me know if need a PR

**Comment by patrickhlauke** [comment]
Fixed (went with a very short "Search for..." `aria-label`, as I think it's clear from context that you're searching within the current documentation site). However, this made me realise that the search has more fundamental issues...see https://github.com/twbs/bootstrap/issues/21851

**Comment by Lausselloic** [comment]
Yes I agree with the aria label, wich is enough robust for assistive tools. 

+1 for global issues on the search patern, I will try to play with patern and post my ideas onto the dedicaded issue

**Comment by louismaximepiton** [comment]
Sorry to disturb this old issue, but don't you think that a `<label class="visually-hidden"></label>` would be better than an `aria-label` since we'd use the semantic of `<label>`? (I could open a PR on this subject if you want to)

**Comment by patrickhlauke** [comment]
there's no real semantic advantage to using a visually hidden `<label>` over an `aria-label` here though.

**Comment by louismaximepiton** [comment]
Ok, thanks for your quick answer !
